utils/constants.jsx

```diff
@@ -1604,4 +1605,3 @@ t('suggestion.archive');
 t('suggestion.mention.groups');
 
 export default Constants;
-
```

Comment:
Aren't we adding a newline at the end of the files? Or is this an extra line?

Reply:
An extra line was at the end of the file. GitHub will say that there is a newline missing otherwise. Check the end of this file https://github.com/mattermost/mattermost-webapp/blob/7c5a2e3bfdf7bb66ac88642e3b426152bbc1fceb/utils/constants.jsx#L1607 to see the extra line, as opposed to the end of most other files https://github.com/mattermost/mattermost-webapp/blob/7c5a2e3bfdf7bb66ac88642e3b426152bbc1fceb/utils/countries.ts#L11